manta-pay/src/crypto/ecc/arkworks.rs

```diff
@@ -36,6 +36,7 @@ use manta_util::serde::{Deserialize, Serialize, Serializer};
 
 pub use ark_ec::{AffineCurve, ProjectiveCurve};
 pub use ark_r1cs_std::groups::CurveVar;
+use manta_crypto::ecc::{PointAdd, PointDouble};
```

Comment:
Move this non-public `use` import together with the other ones. The ordering of imports is

1. `use`
2. `pub use`
3. `mod`
4. `pub mod`

So non-`pub` before `pub`. Also, non-`feature` before `feature` gated and features in alphabetical order.

Reply:
done